src/style.css

```diff
@@ -1,3 +1,79 @@
 body {
   margin: 0;
+  max-width: 1200px;
+  min-width: 1024px;
+}
+
+.menu {
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 60px;
+}
+
+.img {
+  display: absolute;
+  width: 40px;
+  height: 40px;
+  left: 50px;
+  top: 10px;
```

Comment:
Pay attention to the CSS properties order
https://9elements.com/css-rule-order/